src/sql/workbench/contrib/notebook/browser/cellViews/cellToolbar.component.ts

```diff
@@ -51,14 +52,24 @@ export class CellToolbarComponent {
 
 	ngOnInit() {
 		this.initActionBar();
+		this.model.onCellTypeChanged(cell => {
```

Comment:
Make sure you dispose this on `ngOnDestroy`

Reply:
`onCellTypeChanged` returns a disposable for unregistering the event handler (the arrow function)